api/apps/api/test/business-critical/marxan-run/fixtures.ts

```diff
@@ -42,7 +42,7 @@ export const getFixtures = async () => {
     },
     GivenCostSurfaceTemplateFilled: async () => {
       await request(app.getHttpServer())
-        .get(`/api/v1/scenarios/${scenario}/cost-surface/shapefile-template`)
+        .get(`/api/v1/projects/${scenario}/cost-surface/shapefile-template`)
```

Comment:
Tests where this precondition is used are skipped, plus the endpoint is gone as well, so dunno if we should remove it altogether

Reply:
tbh I'd remove this function and its use in the skipped `Marxan run` test